test/unit_Zonotope.jl

```diff
@@ -76,4 +76,8 @@ for N in [Float64, Rational{Int}, Float32]
     Zred2 = reduce_order(Z, 2)
     @test ngens(Zred2) == 4
     @test order(Zred2) == 2
+
+    # test conversion from hyperrectangular sets
+    convert(Zonotope, Hyperrectangle(N[2.,3.], N[4.,5.]))
+    convert(Zonotope, BallInf(N[5.,3.], N(2.)))
```

Comment:
maybe add `@test Z.center == N[2.,3.]` and `@test diag(Z.generators)  == N[4.,5.]`